pkg/kf/commands/apps/integration_test.go

```diff
@@ -42,9 +42,10 @@ const (
 func TestIntegration_Push(t *testing.T) {
 	checkClusterStatus(t)
 	RunKfTest(t, func(ctx context.Context, t *testing.T, kf *Kf) {
-		appName := fmt.Sprintf("integration-push-%d", time.Now().UnixNano())
+		deleteSpace := setupSpace(ctx, t, kf)
```

Comment:
Could/should this be moved into RunKfTest?

Reply:
Love it! I'll do that.